config/locales/forms/edit_forms/en.yml

```diff
@@ -29,7 +29,7 @@ en:
           operator:
             heading: "Operator details"
             labels:
-              operator_name: "Name"
+              operator_name: "Registered Name"
```

Comment:
@Beckyrose200 is making similar changes to another locale file, please liaise for consistency.